LilitYeghiazaryan/functions_12

```diff
@@ -1,15 +1,19 @@
 #include <iostream>
-void func_3(double a, double b, double*& res)
+#include <vector>
+void func_3(double a, double b, double* res)
```

Comment:
Let's not use pointers unless needed. You have basically 2 choices - a) pass vector by reference so that the `func_3` can change it (`void func_3(double a, double b, std::vector<double>& res`) or which is even better is b) return the vector from the `func_3` (`std::vector<double> func_3(double a, double b)`).

Can you think of a reason why option b is better? For one, it is cleaner in terms of the API - you give me 2 doubles I give you back some number of results. Anything else?